styles/components/_tabbedModal.scss

```diff
@@ -32,6 +32,12 @@
 
   .tabContent {
     max-width: 757px;
+  
+    .cornerTR {
```

Comment:
We don't want this to affect any corner buttons anywhere else in the tabConent.

I would either change this to & > .cornerTR so it only affects direct children, which is probably safe, or give the button a new override class that is only in this style sheet, which is completely safe (ie, the button would have cornerTR and modalContentCornerBtn or something).